splunk-otel-android-volley/build.gradle.kts

```diff
@@ -1,10 +1,8 @@
-import java.net.URI
 import java.time.Duration
 
 plugins {
     id("com.android.library")
-    id("maven-publish")
-    id("signing")
+    id("splunk.publish-conventions")
```

Comment:
Nit: WDYT about calling it `splunk.android-library-conventions`? This plugin does a bit more than just setting up artifact publishing